src/MMALSharp.Processing/Handlers/IMotionCaptureHandler.cs

```diff
@@ -33,6 +33,7 @@ public interface IMotionCaptureHandler
         /// <summary>
         /// Disables motion detection. When configured, this will instruct the capture handler not to detect motion.
         /// </summary>
-        void DisableMotionDetection();
+        /// <param name="disableCallbackOnly">When true, motion detection will continue but the OnDetect callback will not be invoked.</param>
+        void DisableMotionDetection(bool disableCallbackOnly = false);
```

Comment:
If a user calls this method and wants to re-enable the callback, how do they do this? Am I right in thinking they just modify `OnDetectEnabled` against a `FrameDiffDriver` instance?

Reply:
Calling `EnableMotionDetection` will re-enable the callback. I will add that to the parameter docs.